lib/runtime/juttle-utils.js

```diff
@@ -50,7 +50,12 @@ function parseTime(points, timeField, context) {
         }
 
         var time = pt[timeField || 'time'];
-        if (time) { pt.time = time; }
+        if (time) {
+            pt.time = time;
+            if (timeField !== 'time') {
```

Comment:
The `if (time)` is from the existing code and I merely am checking that the `timeField` value is not `time` before I delete it from the data. That check on time should be something to fix in another PR where we go through all the code and decide how do we want to check for time not being set which I think @demmer and @dmajda already have some ideas on this matter.